ifrs17/Constants/Validations.ipynb

```diff
@@ -184,7 +184,9 @@
                 "public static string Get (Warning w, params string[] s) => (w, s.Length) switch {",
                 "\n    // Import",
                 "\n    (Warning.ActiveDataNodeWithCashflowBOPI , 1) => $\"Cash flow with AoC Type: {AocTypes.BOP} and Novelty: {Novelties.I} for Group of Contract {s[0]} is not allowed because previous period data are available.\",",
-                "\n    (Warning.ScenarioReCalculations         , 1) => $\"The import of the current file for the Best Estimate scenario makes the result of dependent Scenarios out of date. Hence, the following Scenarios are recalculated: {s[0]}.\", ",
+                "\n    (Warning.VariablesAlreadyImported       , 0) => $\"The import of the current file does not contain any new data. Hence, no data will be saved or calculations will be performed.\",",
```

Comment:
Typo? "... be saved or calculations ..." -->  "... be saved nor calculations ..."

Reply:
Minor. We can capture this in an issue and tackle it in the next release.